plutus-benchmark/ed25519-throughput/Main.hs

```diff
@@ -0,0 +1,213 @@
+-- editorconfig-checker-disable-file
```

Comment:
I think we probably want to avoid introducing more of this :)

Reply:
Oh, I thought we had to put that in.  How do we decide if it's necessary or not then?

Reply:
Hmph, I thought I had documetned this in `CONTRIBUTING` but apparently not. You only need this if `editorconfig-checker` complains at you and you're unwilling to fix it. I mostly put it in to grandfather in old files so I wouldn't have to fix literally everything.